cucumber/authenticators_azure/features/authn_azure_performance.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Azure Authenticator performance
+
```

Comment:
If you can describe the feature like you did before it will be great. It was very clear.